metrics/prometheus-agent/templates/prometheus.yaml

```diff
@@ -8,7 +8,6 @@ metadata:
     app.kubernetes.io/name: {{ template "prometheus-agent.name" . }}
     app.kubernetes.io/component: prometheus
     release: {{ $.Release.Name | quote }}
-    self-monitor: {{ .Values.prometheus.serviceMonitor.selfMonitor | quote}}
```

Comment:
why do we remove it. ?

Reply:
Because I added by mistake on the previous PR